src/BarChart/BarChartHorizontal.story.tsx

```diff
@@ -64,6 +64,54 @@ export const Simple = () => (
   />
 );
 
+export const BarTargetMarker = () => {
+  const sampleData = [
+    {
+      key: 'DLP',
+      data: 10,
+      target: 15
+    },
+    {
+      key: 'Malware',
+      data: 30,
+      target: 25
+    },
+    {
+      key: 'IDS',
+      data: 20,
+      target: 23
+    },
+    {
+      key: 'Phishing Attack',
+      data: 40,
+      target: 45
+    }
+  ];
+
+  return (
+    <BarChart
+      width={500}
+      height={350}
+      data={sampleData}
+      xAxis={<LinearXAxis type="value" />}
+      yAxis={
+        <LinearYAxis
+          type="category"
+          tickSeries={<LinearYAxisTickSeries tickSize={20} />}
+        />
+      }
+      series={
+        <BarSeries
+          colorScheme={'cybertron'}
+          layout="horizontal"
+          padding={0.1}
+          bar={<Bar gradient={Bar.defaultProps.gradient} guide={null} />}
```

Comment:
Why do you have to pass gradient like this?

Reply:
not sure why it uses as it,
it was just copied from above story, difference only by data

Reply:
ok lets remove that.